data/core/units/monsters/Raven.cfg

```diff
@@ -0,0 +1,118 @@
+#textdomain wesnoth-units
+[unit_type]
+    id=Raven
+    name= _ "Raven"
+    race=bird
+    image="units/monsters/raven/raven.png"
+    profile="portraits/monsters/raven.webp"
+    {TRAIT_FERAL_MUSTHAVE}
+    hitpoints=18
+    movement_type=lightfly
+    movement=7
+    experience=24
+    level=0
+    alignment=neutral
+    advances_to=null
+    {AMLA_DEFAULT}
+    undead_variation=falcon
```

Comment:
The Walking Corpse and Soulless variations should become birds too.

Reply:
Well, if your point is about consistency of IDs, yes. However, note that this does _not_ reference the race ID, so it's not strictly necessary to change it. At the same time, it quite likely has similar compatibility issues to change it.